stories/Colors/Colors.jsx

```diff
@@ -1,114 +0,0 @@
-import React from 'react';
```

Comment:
these are still valid components for use in storybook, just need to switch the styling to scss